### PR TITLE
added API for cbplan request content

### PR DESCRIPTION
**Pull request by ravisaurav-tarento**
HI @karthik-tarento kindly review this PR

### CODE REVIEW

File: src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java [hunk 1]
```diff
@@ -509,6 +509,37 @@ public SBApiResponse getCBPlanListForUser(String userOrgId, String authUserToken
         return response;
     }
 
+    public SBApiResponse requestCbplanConten(SunbirdApiRequest request, String userOrgId, String token){
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java]: Follow the existing implementation style.
check userId is not empty - if empty return 400 error
have a validation method to check competency and providerList is not null from the request - if either is null return 400 error

File: src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java [hunk 2]
```diff
@@ -509,6 +509,37 @@ public SBApiResponse getCBPlanListForUser(String userOrgId, String authUserToken
...
+        try{
+            String userId = validateAuthTokenAndFetchUserId(token);
+            Map<String, String> mdoInfo = userUtilityService.getUserDetails(Collections.singletonList(userId), new ArrayList<>()).get(userId);
+            String mdoOrgId = mdoInfo.get(Constants.ROOT_ORG_ID);
+            String competencyJsonString = mapper.writeValueAsString(competency);
+
+            Map<String, Object> propertiesMap = new HashMap<>();
+            propertiesMap.put(Constants.ORG_ID, mdoOrgId);
+            propertiesMap.put(Constants.ID, UUID.randomUUID().toString());
+            propertiesMap.put(Constants.COMPETENCY_INFO , Collections.singletonList(competencyJsonString));
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java]: can't we simply use competency list object ?

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java (hunk 1 above)
Comment (ravisaurav-tarento) [reply src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java]: This is implemented please verify.

File: src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java (hunk 2 above)
Comment (ravisaurav-tarento) [reply src/main/java/org/sunbird/cbp/service/CbPlanServiceImpl.java]: This is updated, we are storing value as an object.

**Pull request merged**